packages/core/src/application.ts

```diff
@@ -139,6 +136,21 @@ export class Application extends Context {
     return this.bind(`routes.${route.verb} ${route.path}`).to(route);
   }
 
+  api(spec: OpenApiSpec) {
+    for (const path in spec.paths) {
+      for (const verb in spec.paths[path]) {
```

Comment:
I am adding two tests for this. Interestingly enough, for-in operator handles both `undefined` and `null` and simply terminates the loop.